CHANGES.txt

```diff
@@ -44,6 +44,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Ninja: Fix issue where Configure files weren't being properly processed when build run
       via ninja.
     - Added ninja mingw support and improved ninja CommandGeneratorAction support.
+    - Added ninja automatic detection of the need for response files
```

Comment:
How about "Update ninja file generation to only create response files for build commands which exceed MAXLINELENGTH" ?

Reply:
done 
0c82b37